examples/yank-yaml-cookbook/all-options.yaml

```diff
@@ -27,12 +27,12 @@ options:
                                                 # with the constraint of being at a distance greater than
                                                 # randomize_ligand_close_cutoff from the receptor.
   temperature: 298.0 * kelvin                   # Temperature of the system.
-  pressure: 1.0 * atmosphere                    # Pressure of the system.
+  pressure: 1.0 * atmosphere                    # Pressure of the system. Set to null for NVT ensemble.
   hydrogen_mass: 1.0 * amu                      # Hydrogen mass for HMR simulations.
-  constraints: HBonds                           # Constrain bond lengths and angles. Possible values are None,
+  constraints: HBonds                           # Constrain bond lengths and angles. Possible values are null,
```

Comment:
To be more clear
```yaml
constraints: None
```
would be interpreted as the string `'None'` by YAML and `schema` will reject it.